src/main/jbake/templates/index.ftl

```diff
@@ -5,55 +5,62 @@
-                        <h4>Metric Data Visualization</h4>
-                        <p>View your data and add charts where you need them most, including real time monitoring. <a href="/overview/index.html#_metric_data_visualization">More...</a></p>
-                    </li>
-                    <li class="col-md-4 col-sm-4">
-                        <a href="/overview/index.html#_application_performance_management"><i class="fa fa-briefcase"></i></a>
-                        <h4>Application Performance Management</h4>
-                        <p>Capture information about the application invocations being executed across your servers. <a href="/hawkular-apm/index.html">More...</a></p>
-                    </li>
-                    <li class="col-md-4 col-sm-4">
-                        <a href="http://manageiq.org/"><img src="http://avatars0.githubusercontent.com/ManageIQ?&s=48" alt="ManageIQ logo"></img></a>
-                        <h4>Integration with ManageIQ</h4>
-                        <p>ManageIQ lets you manage container, virtual, private, and public cloud infrastructures. <a href="http://manageiq.org/">More...</a></p>
-                    </li>
-                </ul>
-            </div>
-        </div>
-    </section>
+</div>
+
+<div class="container">
+  <div class="row">
+    <h2>What is Hawkular?</h2>
+    <div class="paragraph">
+      <p>
+        Hawkular a hawk with a monocular. Hawks are known to have a very sharp vision and very good hunters, they can catch preys anticipating their movements at a very fast speed.
+        The analogy is our goal to be able to monitor and catch anomalies in fast pace environments.
+      </p>
+    </div>
+    <div class="paragraph">
+      <p>
+        The organization started around the end of 2014 as a successor of the <a href="http://www.rhq-project.org">RHQ Project</a>.
+        It’s a set of open source (Apache License v2) components targeted for monitoring solutions and is sponsored by <a href="http://www.redhat.com">Red Hat</a>.
+      </p>
+    </div>
+  </div>
+
+  <div class="row">
+    <h2>Mission Statement and Strategy</h2>
+    <div class="paragraph">
+      <p>
+        <strong>Mission Statement</strong>: Be the most popular open source monitoring solution.
+      </p>
+    </div>
+    <div class="paragraph">
+      <p>
+        The main goal of the Hawkular organization is provide tools to monitor, view, predict, and guide to make informed decisions about deployed applications and infrastructure.
+      </p>
+    </div>
+  </div>
+
+  <div class="row">
+    <h2>Contributing</h2>
+    <div class="paragraph">
+      <p>
+        All contributions are welcome and there are several ways to join the Hawkular community. The best option to reach the developers and other community members. For a complete
+      </p>
+    </div>
+    <div class="paragraph">
+      <p>
+        TODO: another paragraph here
```

Comment:
How Can I Contribute?
There are several ways to join the Hawkular community and contribute to the project.  It’s always a good idea to talk to the current contributors and learn about what’s going on before you get started.
Join the Community
The Hawkular development team can be found on Internet Relay Chat (IRC) at irc://irc.freenode .net/#hawkular (If you can’t use an IRC client on your machine you can find a Web-client here).

E-mail discussions happen on the hawkular-dev mailing list.  To join the mailing list visit the list manager page at https://lists.jboss.org/mailman/listinfo/hawkular-dev and subscribe to the list.

Commit messages are sent to the hawkular-commits mailing list. You can subscribe a thttps://lists.jboss.org/mailman/listinfo/hawkular-commits.
Give Project Feedback
Download one or more of the Hawkular projects and give us feedback on how it works and what you’d like to see in future releases.

Hawkular issues, feature requests, and ideas are captured in the Hawkular Jira instance at JBoss.org. There are several individual trackers for the various sub-projects.  When in doubt about which project to use, just use the Hawkular project and we will re-assign the submissions.
Contribute Code
All of the Hawkular project code lives on GitHub under https://github.com/hawkular. To contribute, fork the respective project(s), implement your changes and open a pull-request. See the developer guide for more information.
Contribute Documentation
Just like code, the Hawkular documentation also lives in repositories on GitHub.  The Hawkular web site can be found  at https://github.com/hawkular/hawkular.github.io. And as with the code, if you want to contribute, fork the repo, create your changes and open a pull-request.